.github/workflows/build-docker-image.yml

```diff
@@ -0,0 +1,33 @@
+name: Build Docker image
+
+on:
```

Comment:
@diox further clarification. This job only runs on workflow_dispatch, meaning you have to manually trigger it. It also only pushes if you explicitly opt in and the default value is false.